webextension/scripts/utils.js

```diff
@@ -443,19 +443,27 @@ function opener(url, option, callback) {
       h = Math.floor(screen.height * 0.9)
     }
     chrome.windows.create({ url: url, width: w, height: h, top: 0, left: 0, type: 'popup' }, (window) => {
-      if (callback) { callback(window.tabs[0].id) }
+      if (callbackFn) { callbackFn(window.tabs[0].id) }
     })
   }
 }
 
-function notify(message, callback) {
+function notify(message, callbackFn) {
   let options = {
     type: 'basic',
     title: 'WayBack Machine',
     message: message,
     iconUrl: chrome.runtime.getURL('images/app-icon/app-icon96.png')
   }
-  chrome.notifications && chrome.notifications.create(options, callback)
+  chrome.notifications && chrome.notifications.create(options, callbackFn)
+}
+
+function callback() {
+  if (chrome.runtime.lastError) {
+      console.log(chrome.runtime.lastError.message);
+  } else {
+      // Tab exists
```

Comment:
This callback is not only about the tab existing or not, correct? It applies to _any_ `lastError`.

Reply:
I think this comment is inaccurate. The correct comment would read:
`// No error occurred`